resources/styles/organisms/site-footer/_index.scss

```diff
@@ -15,32 +15,57 @@
         align-items: start;
     }
 
-    &__links {
-        grid-column: 5 / 11;
-        justify-self: center;
+
+    &__identity {
+        display: grid;
+        grid-column: main;
+        row-gap: config.$thin-h-space;
+
+        @include at(lg) {
+            grid-column: 2 / 5;
+        }
     }
 
     &__brand {
-        grid-column: 2 / 5;
         align-items: center;
-        color: config.$primary;
+        color: config.$yellow-100;
         display: flex;
         gap: config.$thin-h-space;
         padding: config.$thin-v-space 0;
     }
 
     &__socialLinks {
-        grid-column: 11 / 14;
-        justify-self: end;
         display: flex;
         gap: config.$thin-h-space;
     }
 
-    &__utility {
+    &__links {
+        display: grid;
         grid-column: main;
+        row-gap: config.$thin-h-space;
+
+        @include at(lg) {
+            column-gap: config.$h-space;
```

Comment:
For this, I think we should use `var(--column-gap)` instead - it'll make the gaps match here to the typical grid column gaps, which looks more right to me than to have these gaps smaller and fixed while the others are responsible.

Reply:
i used var(--column-gap)